container.go

```diff
@@ -0,0 +1,342 @@
+package hcsshim
+
+import (
+	"encoding/json"
+	"syscall"
+
+	"errors"
+	"github.com/Sirupsen/logrus"
+	"time"
+)
+
+type hcsShimContainer struct {
+	handle HcsSystem
+}
+
+// CreateContainer creates a new container with the given configuration but does not start it.
+func CreateContainer(id string, c *ContainerConfig) (Container, error) {
+	title := "HCSShim::CreateContainer"
+	var (
+		handle   HcsSystem
+		identity syscall.Handle
+		resultp  *uint16
+	)
+
+	configurationb, err := json.Marshal(c)
+	if err != nil {
+		return nil, err
+	}
+
+	configuration := string(configurationb)
+
+	if err := hcsCreateComputeSystem(id, configuration, identity, &handle, &resultp); err != nil {
```

Comment:
I imagine that this will need to support both cases, is there a build number that it was added in so that this change doesn't break internal testing?